client/src/modules/dashboard/Chapters/pages/ChapterPage.tsx

```diff
@@ -29,19 +29,43 @@ export const ChapterPage: NextPage = () => {
     <Layout>
       <Card className={styles.card}>
         <ProgressCardContent loading={loading}>
-          <Heading as="h5" fontWeight="normal">
+          <Heading
+            fontSize={'md'}
+            as="h1"
+            fontWeight="semibold"
+            marginBlock={'2'}
+          >
             {data.chapter.name}
           </Heading>
           <Box>
-            <Link href={`${chapterId}/users`} target="_blank">
+            <Link
+              href={`${chapterId}/users`}
+              target="_blank"
+              paddingBlock={'2'}
+            >
               Chapter Users
             </Link>
           </Box>
-          <HStack>
-            <LinkButton size="sm" href={`${chapterId}/new-event`}>
+          <HStack mt={'2'}>
+            <LinkButton
+              background={'gray.85'}
+              color={'gray.05'}
+              _hover={{
+                background: 'gray.45',
```

Comment:
I had to use high background, because the transition was uncomfortable to my eyes. 

that being said, this doesn't feel like the best approach, I am open to suggestions